spring-cloud-aws-secrets-manager-config/src/main/java/org/springframework/cloud/aws/secretsmanager/AwsSecretsManagerPropertySourceLocator.java

```diff
@@ -78,56 +78,20 @@ public AwsSecretsManagerPropertySourceLocator(AWSSecretsManager smClient, AwsSec
 
 		ConfigurableEnvironment env = (ConfigurableEnvironment) environment;
 
-		String appName = properties.getName();
-
-		if (appName == null) {
-			appName = env.getProperty("spring.application.name");
-		}
+		AwsSecretsManagerPropertySources sources = new AwsSecretsManagerPropertySources(properties, logger);
```

Comment:
As far as I can see you have ignored checking `spring.application.name` which potentially will break users using old way of using Secrets Manager.